features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/org.wso2.carbon.apimgt.core.default.json

```diff
@@ -40,6 +40,9 @@
   "apim.key_manager.pool.max_idle": "100",
   "apim.key_manager.pool.init_idle_capacity": "50",
   "apim.key_manager.key_validation_handler_impl": "org.wso2.carbon.apimgt.keymgt.handlers.DefaultKeyValidationHandler",
+  "apim.monetization.publish_duration": "1",
```

Comment:
1m or 1s

Reply:
This is publish duration in days, which corresponds to config  `<PublishTimeDurationInDays>`.

Reply:
can't we not to maintain a new config for that and keep in different way

Reply:
@tharindu1st example?

Reply:
value should be 1d then.and in unit-resolver.json should have a mapping

Reply:
Fixed with e2aab61